jax/experimental/jax2tf/tests/jax2tf_limitations.py

```diff
@@ -1478,7 +1478,7 @@ def custom_assert(tst, result_jax, result_tf, *, args, tol, err_msg):
     # TODO(frostig): Don't need this conditional once we always
```

Comment:
We can remove this comment, or change it to one that says that we're handling both new- and old-style keys here.

Reply:
Done